molecule/dynamic/group_vars/all

```diff
@@ -0,0 +1,13 @@
+---
+pulp_default_admin_password: password
+prereq_roles:
```

Comment:
We should just call them "roles" or "pulp_roles". Since they aren't prerequisites, they are the meat of the installer.